### PR TITLE
Header - mobile

### LINKED ISSUE #40: Implement header - mobile
Ensure that the header is responsive and works on smaller screens than the desktop implementation

## Acceptance Criteria

* Header works on mobile devices as designed in Figma
* Header continues to work on larger screens
* Header changes responsively as the browser/screen changes
* Design sign-off

### PR DESCRIPTION
**Pull request by MariamM110**
closes #40 

- When the viewport width is less than 768px, the layout changes to mobile for header

### CODE REVIEW

**Comment by SociableSteve** [comment]
Linter says no! 😢

**Comment by SociableSteve** [comment]
You have multiple elements with the same ID. ID's need to be unique. These should probably be classes instead.

File: layouts/components/header.html [hunk 1]
```diff
@@ -12,14 +12,15 @@
...
     </div>
 
-    <nav class="header-desktop">
+    <nav class="header-nav">
       <div class="nav-links">
-        <a class="format-bold" href="/about/">About</a>
-        <a class="format-bold" href="/developers/">Developers</a>
-        <a class="format-bold" href="/help/">Learn</a>
+        <a class="format-bold mobile-link" href="/about/">About</a>
+        <a class="format-bold mobile-link" href="/developers/"
```
Comment (SociableSteve) [inline layouts/components/header.html]: The formatting 😬 Can you keep the tag close on the same line as the tag open where properties are not split over multiple lines please?

**Review by SociableSteve — CHANGES_REQUESTED** [review]

**Comment by SociableSteve** [comment]
Looks nice but the content is still visible underneath and can be scrolled while in the menu.

image

**Comment by SociableSteve** [comment]
image

Some kind of line is now crossing the logo 😬 this might gonna get annoying

**Review by bp289 — APPROVED** [review]

**Pull request merged**